MIP0/mip0.md

```diff
@@ -7,287 +7,249 @@ Title: The Maker Improvement Proposal Framework
--   A MIP has been replaced by a newer, more updated MIP Proposal.
--   A MIP no longer makes sense to keep in consideration.
-    
-  
-**MIP Status Change Process:**
-    
+If a MIP Author requests a status change for a MIP, a MIP Editor will review it. If the status change is warranted, the MIP Editor will change the status. Otherwise, the MIP Editor will revert and highlight issues for the MIP Author to fix before requesting another status change.
 
-A status change for a MIP is requested by the MIP Author and will be reviewed by the MIP Editor(s) to see if it meets the status criteria of the requested status change. If it does, the Editor(s) will change the status of the MIP and the Author may proceed with the next stage of the process. If it does not, the MIP Editor(s) will revert with highlighted issues, and the Author must fix the highlighted issues before requesting another status change.
-    
 ---
 ### MIP0c4: MIP Components and MIP Component Types
 
 
-**MIP Components**
+#### MIP Components
+
+In order to satisfy the Core Principles of clarity and completeness, a MIP may need to have multiple components that each define distinct units of logic.
+
+> MIP Components follow the naming convention `MIPXcY`, where `X` is the parent MIP and `Y` is the component number.
+
+##### Component Types
+
+A MIP Component can be of one of two types: General or Process. The type of General MIP Components is usually left undeclared: It is the assumed type of components that are not of Process type.
+
+#### Process MIP Component  
+
+MIP Components of the Process type shape a specific process flow for the Maker community to adopt and standardize with respect to how governance operates. This MIP component type helps streamline particular processes in a transparent, open, and traceable manner.
+
+In this MIP, `MIP0c6`, `MIP0c12` and `MIP0c13` are Process Components.
+
+Process MIP Components require no special template.
+
+#### Subproposals
+
+Subproposals instantiate the processes defined by Process MIP Components. For example, every concrete onboarding of a Governance Facilitator requires creating a subproposal off the Process MIP Component `MIP0c12`.
+
+> Subproposals follow the naming convention `MIPXcY-SPZ`, where `X` is the parent MIP, `Y` is the component number, and `Z` is the subproposal number.
 
-- When necessary, MIPs can have multiple components if it needs to contain multiple units of logic to satisfy completeness. A MIP can also have only a single component.
-- MIP components are categorized by types, depending on what kind of logic they contain. MIP components are named by their parent MIP. The abbreviation convention MIPXcY is used to refer to these components (as seen in this document).
-- A MIP component has one type or no types. 
+Subproposals of Process MIP Components with additional MIP Component types inherit the same types.
```

Comment:
Maybe just delete this? I'm not even sure what it's supposed to mean.

Reply:
Hahaha. Same here